modules/rgbd/src/tsdf.cpp

```diff
@@ -1121,7 +1206,7 @@ TSDFVolumeGPU::TSDFVolumeGPU(float _voxelSize, cv::Matx44f _pose, float _raycast
                              Point3i _resolution) :
     TSDFVolume(_voxelSize, _pose, _raycastStepFactor, _truncDist, _maxWeight, _resolution, false)
 {
-    volume = UMat(1, volResolution.x * volResolution.y * volResolution.z, CV_32FC2);
+    volume = UMat(1, volResolution.x * volResolution.y * volResolution.z, CV_8UC2 /*CV_32FC2*/);
```

Comment:
Please remove that extra comment, it's confusing like `CV_32FC2` is a valid alternative

Reply:
done